tfjs-converter/python/tensorflowjs/converters/tf_saved_model_conversion_v2.py

```diff
@@ -323,7 +278,7 @@ def extract_weights(graph_def,
                   initializer_signature_def=initializer_signature_def,
                   resource_ids_maps=resource_ids_maps,
                   metadata=metadata)
-
+ """
```

Comment:
remove comments?

Reply:
Thanks! Sorry, I should have proof-read this better.